README.md

```diff
@@ -0,0 +1,2 @@
+# Jupyter notebooks with [matchms](https://github.com/matchms/matchms) examples.
+# Please create new issues [here](https://github.com/matchms/matchms/issues).
```

Comment:
I would prefer to omit the second line, and let people decide if their issue needs to be filed against https://github.com/matchms/matchms (if they think it has to do with the matchms library), or against https://github.com/matchms/notebooks (if they think there is an issue with one of the notebooks).